code/run.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# LOAD DEPENDENCIES AND MODULES --------------------------------------
+source /usr/share/lmod/lmod/init/bash
+
+module load base-env
+module load proxy
+module load conda
+conda activate R4.2.3
+conda activate
+module load stata/17.0
+
+# python packages
+pip install numpy
+pip install pandas
+pip install fuzzywuzzy
+pip install xlrd
+pip install openpyxl
+
+# R packages
+Rscript -e "install.packages(c("tidyverse", "haven", "readxl", "broom", "sf", "rnaturalearthdata", "rnaturalearth", "cowplot", "scales", "biscale", "dichromat", "geosphere"), repos='https://cran.rstudio.com')"
+
+# SET PATH HERE ------------------------------------------------------
+cd /win/l1werpfile2/shared/Cheikh/Research/Ishan/Water/pp_replication
```

Comment:
@cheikhfall07 the README says you should edit line 19 of `run.sh` to direct to your local filepath. did you mean that to be line 24?

Reply:
Yes, I updated the script without comitting the changes! But it should all be consistent now!